netket/experimental/operator/_fermion_operator_2nd.py

```diff
@@ -231,9 +241,37 @@ def from_openfermion(
     def __repr__(self):
         return (
             f"FermionOperator2nd(hilbert={self.hilbert}, "
-            f"n_operators={len(self._operators)}, dtype={self.dtype})"
+            f"n_operators={len(self.terms)}, dtype={self.dtype})"
         )
 
+    @property
+    def terms(self) -> OperatorTermsList:
+        return self._terms
+
+    @property
+    def weights(self) -> OperatorWeightsList:
+        return self._weights
+
+    @property
+    def constant(self) -> Union[float, complex]:
```

Comment:
please also add a docstring

```suggestion
    def constant(self) -> Number:
```